apps/codeforafrica/src/components/AboutChildPageHeader/AboutChildPageHeader.js

```diff
@@ -69,7 +69,7 @@ const AboutChildPageHeader = React.forwardRef(
               <MemberFigureRoot
                 {...FigureProps}
                 sx={{
-                  background: `url(${image.src})`,
+                  background: `url(${image.url})`,
```

Comment:
Can we standardise on image.src?

Reply:
Didn't quite get what you meant here @kilemensi. Does your proposal imply adding a `src` field to Media?